RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -340,6 +341,13 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
      fitParsVec.push_back(0.);
      fitParsVec.push_back(999.);
      fitParsVec.push_back(false);
+   } else {
```

Comment:
It looks like the old behavior is restored this way: old version (pre10) passed a vector of parameters of size 1 on output and in this case the default settings (time = -9999 and ampl = 0) were used. OK.

Note that this method is called with ts4Min == 0 and it looks like we end up in this last "else" only if tsTOTen == 0.
Is this case singled out for a reason or was it an old bug in the method?